package.json

```diff
@@ -31,7 +37,8 @@
 	},
 	"husky": {
 		"hooks": {
-			"pre-commit": "pretty-quick --staged"
+			"pre-commit": "pretty-quick --staged",
+			"commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
```

Comment:
*really* not sure how annoying this would be as a git hook. Having our CI builds run  this linting check is probably good enough I'm guessing?